apps/storybook-react/vite.config.ts

```diff
@@ -1,14 +1,6 @@
-// import path from 'path';
+import react from '@vitejs/plugin-react';
```

Comment:
Updates vite config to be aligned with react https://github.com/vitejs/vite-plugin-react/tree/main/packages/plugin-react#vitejsplugin-react-